tests/foreman/cli/test_host.py

```diff
@@ -70,1769 +64,1491 @@
+):
+    host = make_fake_host(
+        {
+            'content-source-id': module_default_proxy['id'],
+            'content-view-id': module_published_cv.id,
+            'lifecycle-environment-id': module_lce_library.id,
+            'organization': module_org.name,
+        }
+    )
+    yield host
+    Host.delete({'id': host['id']})
+
+
+@pytest.fixture(scope="module")
+def module_make_ak(module_lce, module_org, module_promoted_cv):
+    return make_activation_key(
+        {
+            'content-view-id': module_promoted_cv.id,
+            'lifecycle-environment-id': module_lce.id,
+            'organization-id': module_org.id,
+        }
+    )
+
+
+@pytest.fixture(scope="function")
+def function_make_host(module_default_proxy):
+    host_template = entities.Host()
+    host_template.create_missing()
+    org_id = host_template.organization.id
+    loc_id = host_template.location.id
+    # using CLI to create host
+    return make_host(
+        {
+            'architecture-id': host_template.architecture.id,
+            'domain-id': host_template.domain.id,
+            'environment-id': host_template.environment.id,
+            'location-id': loc_id,
+            'mac': host_template.mac,
+            'medium-id': host_template.medium.id,
+            'name': host_template.name,
+            'operatingsystem-id': host_template.operatingsystem.id,
+            'organization-id': org_id,
+            'partition-table-id': host_template.ptable.id,
+            'puppet-proxy-id': module_default_proxy['id'],
+            'root-password': host_template.root_pass,
+        }
+    )
+
+
+@pytest.fixture(scope="function")
+def function_make_user(function_make_host):
+    """
+    Returns dict with user object and with password to this user
+    """
+    user_name = gen_string('alphanumeric')
+    user_password = gen_string('alphanumeric')
+    org = Org.info({'name': function_make_host['organization']})
+    location = Location.info({'name': function_make_host['location']})
+    user = make_user(
+        {
+            'admin': False,
+            'default-organization-id': org['id'],
+            'organization-ids': [org['id']],
+            'default-location-id': location['id'],
+            'location-ids': [location['id']],
+            'login': user_name,
+            'password': user_password,
+        }
+    )
+    return {'user': user, 'password': user_password}
+
+
+# -------------------------- CREATE SCENARIOS -------------------------
+@pytest.mark.host_create
+@pytest.mark.tier1
+@pytest.mark.upgrade
+def test_positive_create_and_delete(module_lce_library, module_published_cv):
+    """A host can be created and deleted
+
+    :id: 59fcbe50-9c6b-4c3c-87b3-272b4b584fb3
+
+    :expectedresults: A host is created and deleted
+
+    :BZ: 1260697, 1313056, 1361309
+
+    :CaseImportance: Critical
+    """
+    name = valid_hosts_list()[0]
+    host = entities.Host()
+    host.create_missing()
```

Comment:
isn't this same as `entities.Host().create()`? I think the `create` method supports `create_missing` parameter, which used to be set to `True` by default